lib/utils.js

```diff
@@ -129,6 +129,17 @@ exports.equals = function(actual, expected, ignoredKeys){
   }
 };
 
+exports.contains = function(actual, expected){
+  for (var key in expected) {
+    if (actual[key] !== expected[key]) {
```

Comment:
Query strings can result in nested objects (e.g. `a[b]=4` makes `{ a: { b: 4 } }`), so you should probably use `exports.equals` here